tests/components/airpatrol/test_config_flow.py

```diff
@@ -0,0 +1,180 @@
+"""Test the AirPatrol config flow."""
+
+from airpatrol.api import AirPatrolAuthenticationError, AirPatrolError
+
+from homeassistant import config_entries
+from homeassistant.components.airpatrol.const import DOMAIN
+from homeassistant.const import CONF_ACCESS_TOKEN, CONF_EMAIL, CONF_PASSWORD
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import FlowResultType
+
+from tests.common import MockConfigEntry
+
+TEST_USER_INPUT = {
+    CONF_EMAIL: "test@example.com",
+    CONF_PASSWORD: "test_password",
+}
+
+
+async def test_async_step_reauth_confirm_success(
```

Comment:
Can we add a test where we reauth with a different account and get a non-matching unique id? It should abort as its a different account